gpustat/core.py

```diff
@@ -326,15 +327,21 @@ def full_process_info(p):
         if processes is None:
             # None (not available)
             reps += ' ({})'.format(NOT_SUPPORTED)
-        else:
+        elif not no_processes:
             for p in processes:
                 reps += ' ' + process_repr(p)
                 if show_full_cmd:
                     full_processes.append(os.linesep + full_process_info(p))
+
+        # Remove trailing | if no process information is shown.
+        if no_processes and reps[-1]=="|":  
+            reps = reps[:-2]
```

Comment:
Sorry to nitpick this but please follow the conventional styles. Trailing spaces should be stripped also. In addition, if you want to strip like `reps[:-2]` then `.endswith` check should be with `" |"`, not `"|"`.

Reply:
Great, removed the stripping altogether, and now instead added the " |" conditionally.